init-tools.msbuild

```diff
@@ -4,6 +4,9 @@
     <EnableDefaultItems>false</EnableDefaultItems>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)Tools/$(BuildToolsPackageVersion)</BaseIntermediateOutputPath>
+    <FeedTasksVersion>1.0.0-prerelease-02011-01</FeedTasksVersion>
+    <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage>
+    <PackageTargetFallback>$(PackageTargetFallback);netframework</PackageTargetFallback>
```

Comment:
what is the netframework packagetargetfallback for? that isn't even a valid TFM

Reply:
Restore of Microsoft.DotNet.Build.Tasks.Feed package fails without this - I'm guessing there's something wonky with the package, because other valid TFMs didn't work either.

Reply:
@karajas did you get a chance to fix the package to use a property TFM?